packages/converters/src/to-boolean/index.test.ts

```diff
@@ -10,34 +10,25 @@
  * governing permissions and limitations under the License.
  */
 
-import { expect, it, describe } from 'vitest';
-import { toBoolean } from './';
+import { describe, expect, it } from 'vitest';
 
-const truthy = [1, '1', 'on', 'true', 'yes', true, 'ON', 'YES', 'TRUE'];
-const falsey = [
-  0,
-  '0',
-  'off',
-  'false',
-  'no',
-  false,
-  [],
-  {},
-  'OFF',
-  'NO',
-  'FALSE',
-];
+import {
+  createFixture,
+  falsey,
+  truthy,
+} from '../../../constants/src/__fixtures__/boolean';
```

Comment:
I am on the fence on this one for a few reasons:

1. We are bridging packages somewhat _incorrectly_ here, if you are referencing a package import like `@accelint/constants` this means you are technically referencing "compiled" artifacts not raw source code. 
2. The line in the sand on what does or does not constitute a "shared" or "constant" fixture will become muddy over time (especially as newer folks onboard). These checks are so incredibly simple I don't mind some minor copy paste.
3. If we absolutely must have a centralized spot for fixtures, which is something we do on one of our projects, we should just make it a formal package e.g. `@accelint/fixtures`. This does have some of the issues raised in point 2 but at the very least allows us to "compile" things like a normal package and import them as such as well.

Some fun reading regarding my thoughts around reuse, copy paste, and abstractions:

- http://programmingisterrible.com/post/139222674273/write-code-that-is-easy-to-delete-not-easy-to
- https://sandimetz.com/blog/2016/1/20/the-wrong-abstraction

Reply:
I think this is largely moot after the refactor of `toBoolean` and `is-noyes` (predicates) lands.

Reply:
Good reads by the way.

Reply:
Removed after the `toBoolean` refactor.